src/commands/utility/userinfo.js

```diff
@@ -13,23 +13,29 @@ module.exports = class UserInfo extends Command {
         full: true,
         required: false,
         acceptBot: true,
-        acceptSelf: true
+        acceptSelf: true,
+        acceptPartial: true
       }]
     }, client)
   }
 
-  run ({ t, guild, member: author, channel, language }, member = author) {
+  async run ({ t, guild, member: author, channel, language }, member = author) {
     moment.locale(language)
-    channel.send(
-      new SwitchbladeEmbed()
-        .setTitle(member.displayName)
-        .setThumbnail(member.user.displayAvatarURL({ format: 'png' }))
-        .addField(t('commands:userinfo.tag'), member.user.tag, true)
-        .addField(t('commands:userinfo.id'), member.id, true)
-        .addField(t('commands:userinfo.status'), t(`commands:userinfo.${member.presence.status}`, { status: this.getEmoji(member.presence.status) }), true)
-        .addField(t('commands:userinfo.createdAt'), `${moment(member.user.createdTimestamp).format('LLL')}\n(${moment(member.user.createdTimestamp).fromNow()})`, true)
-        .addField(t('commands:userinfo.joinedAt'), `${moment(member.joinedTimestamp).format('LLL')}\n(${moment(member.joinedTimestamp).fromNow()})`, true)
+    const isUserPartial = !member.joinedTimestamp
+    const createdAt = isUserPartial ? member.createdAt : member.user.createdTimestamp
+    const embed = new SwitchbladeEmbed()
+      .setTitle(isUserPartial ? member.username : member.displayName)
+      .setThumbnail((isUserPartial ? member : member.user).displayAvatarURL({ format: 'png' }))
+      .addField(t('commands:userinfo.tag'), isUserPartial ? member.tag : member.user.tag, true)
+      .addField(t('commands:userinfo.id'), member.id, true)
+      .addField(t('commands:userinfo.status'), t(`commands:userinfo.${member.presence.status}`, { status: this.getEmoji(member.presence.status) }), true)
```

Comment:
If the user is an actual PartialUser, as in the bot does not share a Guild with them, we will not have this information (and if we do, it will always be "offline")

Reply:
We do have this info, I have not checked if its always offline but it probably is by looking at how the Discord Client behaves when a friend is accepted. I'll look more into it.